.travis.yml

```diff
@@ -37,20 +37,26 @@ matrix:
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
       script:
         - docker-compose run backend ./runtests.sh
+      after_success:
+        - codecov -f coverage.xml -t 995256be-3a1f-4d10-b181-e24f88603b6b
 
     - env: python-tests-main
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
       script:
         # Several security features in settings.py (eg setting HSTS headers) are conditional on
         # 'https://' being in the site URL. In addition, we override the test environment's debug
         # value so the tests pass. The real environment variable will be checked during deployment.
         - docker-compose run -e SITE_URL=https://treeherder.dev -e TREEHERDER_DEBUG=False backend python -bb ./manage.py check --deploy --fail-level WARNING
         # Using `-bb` mode to surface BytesWarnings: https://docs.python.org/3.7/using/cmdline.html#cmdoption-b
-        - docker-compose run backend python -bb -m pytest tests/ --runslow --ignore=tests/selenium/
+        - docker-compose run backend bash -c "pytest --cov=./ --cov-report=xml tests/ --runslow --ignore=tests/selenium"
```

Comment:
I don't believe you need `--cov=./` is that right?

Reply:
It was part of the standard commands that I found everwhere. Let me try without it

Reply:
Not forming the coverage.xml file if we omit ```--cov=./```